pkg/broker/config/volume/volume_test.go

```diff
@@ -168,12 +168,16 @@ func TestSyncConfigFromFile(t *testing.T) {
 	}
 }
 
-func atomicWriteFile(t *testing.T, tempDir, file string, bytes []byte) {
+func atomicWriteFile(t *testing.T, file string, bytes []byte) {
 	t.Helper()
 	// In order to more closely replicate how K8s writes ConfigMaps to the file system, we will
 	// atomically swap out the file by writing it to a temp directory, then renaming it into the
 	// directory we are watching.
-	tmpFile, err := ioutil.TempFile(tempDir, "temp-*")
+	dir, err := ioutil.TempDir("", "temp-*")
+	if err != nil {
+		t.Fatalf("unexpected error from creating temp dir: %v", err)
+	}
```

Comment:
```suggestion
	}
	defer os.RemoveAll(tempDir)
```

Then, if you want, you can remove the defer to delete `tmpFile`.